src/register.coffee

```diff
@@ -126,6 +139,27 @@ module.exports = (options={}) ->
 
             module._compile transformed, fileName
 
+    # setup any custom post processors
+    if _.isArray(options.postProcessors) and options.postProcessors.length
+        options.postProcessors.forEach (processorOpts={}) ->
+            {ext, fn} = processorOpts
+            if !(_.isString(ext) and _.isFunction(fn))
+                return
+            else if "._coffee" is ext and options.streamlinejs
```

Comment:
To check for someone trying to add a require hook for `._coffee`, after the previous block would have already done so, because the user specified `options.streamlinejs = true`, as well. The `streamlinejs = true` hook takes precedence. I could, alternatively, issue warning and let it clobber the previous require hook.